cou/steps/plan.py

```diff
@@ -72,6 +72,13 @@ async def generate_plan(analysis_result: Analysis, backup_database: bool) -> Upg
     print(f"Upgrading cloud from '{analysis_result.current_cloud_os_release}' to '{target}'\n.")
 
     plan = UpgradeStep(description="Top level plan", parallel=False)
+    plan.add_step(
+        UpgradeStep(
+            description="verify that all OpenStack applications are in idle state",
```

Comment:
I prefer keeping the start of the description consistent with capital letter
```suggestion
            description="Verify that all OpenStack applications are in idle state",
```

I also made the change for backup step in #163